hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieFileGroupReader.java

```diff
@@ -127,6 +129,7 @@ private HoodieFileGroupReader(HoodieReaderContext<T> readerContext, HoodieStorag
     HoodieTableConfig tableConfig = hoodieTableMetaClient.getTableConfig();
     this.partitionPath = fileSlice.getPartitionPath();
     this.partitionPathFields = tableConfig.getPartitionFields();
+    this.partialUpdateMode = tableConfig.getPartialUpdateMode();
```

Comment:
for full record merge cases, either this should be Option<PartialUpdateMode> or we need a NONE PartialUpdateMode. 
if not, what would you set the value for this mode in full record merge use-cases.